rwandaemr-content/backend_configuration/roles/roles-stockmanagement.csv

```diff
@@ -0,0 +1,8 @@
+Uuid,Role name,Description,Inherited roles,Privileges
+7d8d214d-2188-11ed-9dff-507b9dea1806,Stock Management Base Role,This Role is the default Role for stock management with basic functionality with less privileges,,App: stockmanagement.dashboard; App: stockmanagement.stockItems; App: stockmanagement.stockoperations; Get Concepts; Get Locations; Get Users; Get Roles; App: stockmanagement.stockSources;App: stockmanagement.stockOperationType; Task: stockmanagement.party.read
+cca4be4b-2188-11ed-9dff-507b9dea1806,Inventory Manager,This enables managing stock management operations accross the stock management application,Stock Management Base Role,Task: stockmanagement.stockItems.mutate; Task: stockmanagement.stockoperations.mutate; Task: stockmanagement.stockoperations.approve; Task: stockmanagement.stockSources.mutate; Task: stockmanagement.stockoperations.receiveitems
+2083fd40-3391-11ed-a667-507b9dea1806,Inventory Administrator,This Role enables full access to stock management application,Inventory Manager,App: stockmanagement.userRoleScopes; Task: stockmanagement.userRoleScopes.mutate
+d210eb66-2188-11ed-9dff-507b9dea1806,Inventory Clerk,This Role enables access to daily stock operations,Stock Management Base Role,Task: stockmanagement.stockoperations.mutate; Task: stockmanagement.stockSources.mutate; Task: stockmanagement.stockoperations.receiveitems
+8ee2f2ac-467f-11ed-8109-00155dcc3fc0,Inventory Provider Access,This Role enables a provider to view inventory items and access quantities available,,App: stockmanagement.stockItems; Get Concepts; Get Locations; Get Users; App: stockmanagement.stockSources; App: stockmanagement.stockOperationType; Task: stockmanagement.party.read; Task: stockmanagement.stockItems.dispense.qty
+84bdd876-4694-11ed-8109-00155dcc3fc0,Inventory Dispensing,This Role enables a provider to dispence inventory items,Inventory Provider Access,Task: stockmanagement.stockItems.dispense
+a49be648-6b0a-11ed-93a2-806d973f13a9,Inventory Reporting,This Role enables a provider to access reports,Stock Management Base Role,App: stockmanagement.reports; Task: stockmanagement.reports.mutate; Manage Scheduler; Get Patients
```

Comment:
These are just example roles, taken from what the refapp is configuring.  We likely don't want all of these as-is.  But we do need these permissions assigned to roles as appropriate.  @nic10ben and @jberchmas please review.